### ISSUE TITLE
Accessibility Issue - Windows Display Scaling is not honoured

### ISSUE THREAD

**Issue by crhultay** [kobolabs/Kobo-Reader#135]
I have a 45" Display set at 3840x2160 with a Scale set to 175% as I have bad eyesight.

![Image](https://github.com/user-attachments/assets/daba1ba6-a940-45d1-bdde-e623c5db5e93)

However, when I run Kobo Desktop the scale is not honoured. Instead, it appears to be displaying at 100% scale.

I'm aware I could reduce my resolution to "solve" the issue, however I use my screen real estate for two applications. When I increase resolution, I lose detail in images and some applications over-side their borders, etc.

![Image](https://github.com/user-attachments/assets/b774d395-bbfc-4d92-b3c5-e3608c209037)

I think this is an accessibility issue that should be remedied.

**Comment by danielatdigital** [comment]
Totally agree.
